files/image_config/monit/memory_checker

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```

Comment:
Add blank line after shebang. Also suggest changing to `#!/usr/bin/env python3` to ensure it is run with Python 3 until we completely eliminate Python 2.

Reply:
@jleveque this PR target is 201911 which does not support python3. So i assume this should be fine.

Reply:
Yes, you're correct. Thank you!

Reply:
Updated!